### ISSUE TITLE
Possible bug with `ember g` with `--integration` flag

### ISSUE THREAD

**Issue by twokul** [ember-cli/ember-cli#6079]
related: https://github.com/ember-cli/ember-cli-legacy-blueprints/pull/37

environment:
- ember-cli: v2.4.2...v2.6.3
- Windows/AppVeyor with Node 5

from @Turbo87

> I narrowed it down a bit. it seems that this specific change in v2.6.3 isn't responsible for the broken test since it already broke in the last GreenKeeper PR for Ember CLI. we are currently using v2.4.2 and using v2.6.2 is broken on Windows/AppVeyor with Node 5, so something between those releases must've changed something. I'll try to reproduce it in a VM and narrow it down further.

```
AssertionError: expected "tests/integration/helpers/foo/bar-baz-test.js" to exist

parent path "tests/integration/helpers/foo" does not exist.
parent path "tests/integration/helpers" does not exist.
parent path "tests/integration" does not exist.
parent path "tests" exists and contains:
- .jshintrc
- helpers/
- index.html
- test-helper.js
- unit/
```

I have just tested ember g helper-test foo/bar-baz --integration on Ember CLI 2.6.2 and indeed there seems to be a problem:

```
The option '--integration' is not registered with the generate command. Run `ember generate --help` for a list of supported options.
installing helper-test
  create tests/unit/helpers/foo/bar-baz-test.js
apparently it is creating a unit test instead of integration test
```

**Comment by trabus** [comment]
> I have just tested ember g helper-test foo/bar-baz --integration on Ember CLI 2.6.2 and indeed there seems to be a problem:

This is due to the current tagged version (v0.1.1) used by ember-cli [not having integration test support yet](https://github.com/ember-cli/ember-cli-legacy-blueprints/blob/v0.1.1/blueprints/helper-test/index.js#L6). We haven't cut a release since April, and the changes to support helper-test integration tests were added after that release.

After testing locally (linking `ember-cli-legacy-blueprints#master` to `ember-cli#2.7.0-beta.6`, using node v5.12.0) I was unable to replicate the issue. I also tested linking `ember-cli#2.7.0-beta.6` to `ember-cli-legacy-blueprints` for its test suite, and was unable to replicate the failing test reported [here](https://github.com/ember-cli/ember-cli-legacy-blueprints/pull/37). I also tested manually updating the blueprint inside an installed version of `ember-cli#2.6.2` inside a project, and it resolved the issue as well.

I think we can solve this by cutting a new release of ember-cli-legacy-blueprints and updating it in ember-cli. We'll need to tag a new release and update it in ember-cli for that to function properly. I've verified by linking locally and running the same command successfully.
